diff --git a/node_modules/@baklavajs/plugin-renderer-vue/dist/index.cjs b/node_modules/@baklavajs/plugin-renderer-vue/dist/index.cjs
index d30cbc0..9971a83 100644
--- a/node_modules/@baklavajs/plugin-renderer-vue/dist/index.cjs
+++ b/node_modules/@baklavajs/plugin-renderer-vue/dist/index.cjs
@@ -4339,16 +4339,16 @@ var EditorView = /** @class */ (function (_super) {
         addNodeSubmenu.push.apply(addNodeSubmenu, (0,tslib__WEBPACK_IMPORTED_MODULE_4__.__spreadArray)([], (0,tslib__WEBPACK_IMPORTED_MODULE_4__.__read)(defaultNodes), false));
         this.contextMenu.items = [
             {
-                label: "Add Node",
+                label: "ノードを追加",
                 submenu: addNodeSubmenu,
             },
             {
-                label: "Copy Nodes",
+                label: "ノードコピー",
                 value: "copy",
                 disabledFunction: function () { return _this.selectedNodes.length === 0; },
             },
             {
-                label: "Paste Nodes",
+                label: "ノード貼付け",
                 value: "paste",
                 disabledFunction: function () { return _this.clipboard.isEmpty; },
             },
diff --git a/node_modules/@baklavajs/plugin-renderer-vue/dist/index.js b/node_modules/@baklavajs/plugin-renderer-vue/dist/index.js
index 460137b..8d26f18 100644
--- a/node_modules/@baklavajs/plugin-renderer-vue/dist/index.js
+++ b/node_modules/@baklavajs/plugin-renderer-vue/dist/index.js
@@ -4349,16 +4349,16 @@ var EditorView = /** @class */ (function (_super) {
         addNodeSubmenu.push.apply(addNodeSubmenu, (0,tslib__WEBPACK_IMPORTED_MODULE_4__.__spreadArray)([], (0,tslib__WEBPACK_IMPORTED_MODULE_4__.__read)(defaultNodes), false));
         this.contextMenu.items = [
             {
-                label: "Add Node",
+                label: "ノードを追加",
                 submenu: addNodeSubmenu,
             },
             {
-                label: "Copy Nodes",
+                label: "ノードコピー",
                 value: "copy",
                 disabledFunction: function () { return _this.selectedNodes.length === 0; },
             },
             {
-                label: "Paste Nodes",
+                label: "ノード貼付け",
                 value: "paste",
                 disabledFunction: function () { return _this.clipboard.isEmpty; },
             },
@@ -5171,8 +5171,8 @@ var NodeView = /** @class */ (function (_super) {
             x: 0,
             y: 0,
             items: [
-                { value: "rename", label: "Rename" },
-                { value: "delete", label: "Delete" }
+                { value: "rename", label: "名前変更" },
+                { value: "delete", label: "削除" }
             ]
         };
         return _this;
