diff --git a/node_modules/@baklavajs/plugin-renderer-vue3/dist/index.js b/node_modules/@baklavajs/plugin-renderer-vue3/dist/index.js
index 0e33f18..dae4b81 100644
--- a/node_modules/@baklavajs/plugin-renderer-vue3/dist/index.js
+++ b/node_modules/@baklavajs/plugin-renderer-vue3/dist/index.js
@@ -1592,16 +1592,16 @@ var EditorView = /** @class */ (function (_super) {
         addNodeSubmenu.push.apply(addNodeSubmenu, Object(tslib__WEBPACK_IMPORTED_MODULE_0__["__spread"])(defaultNodes));
         this.contextMenu.items = [
             {
-                label: "Add Node",
+                label: "ノードを追加",
                 submenu: addNodeSubmenu,
             },
             {
-                label: "Copy Nodes",
+                label: "コピー",
                 value: "copy",
                 disabledFunction: function () { return _this.selectedNodes.length === 0; },
             },
             {
-                label: "Paste Nodes",
+                label: "貼付け",
                 value: "paste",
                 disabledFunction: function () { return _this.clipboard.isEmpty; },
             },
